File: backend/src/main/java/com/NBE_4_5_2/Team5/domain/payment/dto/PaymentMetaData.java
package com.NBE_4_5_2.Team5.domain.payment.dto;

import com.NBE_4_5_2.Team5.domain.payment.entity.Payment;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@Getter
@Setter
@AllArgsConstructor
public class PaymentMetaData {
	private String paymentId;
	private Integer amount;

	public PaymentMetaData(Payment payment) {
		this.paymentId = payment.getId();
		this.amount = payment.getTotalPrice();
	}
}

File: backend/src/main/java/com/NBE_4_5_2/Team5/domain/payment/dto/PaymentMetaData.kt
package com.NBE_4_5_2.Team5.domain.payment.dto

import com.NBE_4_5_2.Team5.domain.payment.entity.Payment
import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter

@NoArgsConstructor
@Getter
@Setter
@AllArgsConstructor
class PaymentMetaData(payment: Payment) {
    private var paymentId = payment.id
    private var amount = payment.totalPrice
}
